akka-stream/src/main/mima-filters/2.6.x.backwards.excludes/26622-stagelogic-internal-api.excludes

```diff
@@ -0,0 +1,2 @@
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.stage.GraphStageLogic.getEagerStageActor")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.stage.GraphStageLogic#StageActor.this")
```

Comment:
Shouldn't this be in the directory 2.6.10.backwards.excludes rather?

Reply:
I wasn't quite sure, I assumed it should be in the directory for release the change will be included?

Reply:
Ah, we did release 2.6.11 last week, so that's the directory it should be in, I forgot that. 

You can think of it as "binary incompabilities (to ignore) compared to this version", so always the last previous release.

Reply:
ok, I can move that accordingly

Reply:
I also moved `29586-sink-internal-api.excludes` to `2.10` as it was released with `2.11`